macros/src/lib.rs

```diff
@@ -193,11 +193,11 @@ fn gen_dispatch(traits: &[syn::Path], ident: &Ident) -> TokenStream2 {
     quote!(
         #[async_trait]
         impl MessageDispatch for #ident {
-            async fn dispatch<'disp__,'ctx__,'msg__>(
-                &'disp__ self,
-                ctx: &'ctx__ Context,
-                message: Message<'msg__>,
-            ) -> std::result::Result<Message<'msg__>, RpcError> {
+            async fn dispatch(
+                &self,
+                ctx: &Context,
```

Comment:
Nit: While we are here, all of these types should have fully qualified paths. So this should be `::wasmbus-rpc::Context` and `Message` should be something similar

Reply:
done